src/frontend/komponenter/Fagsak/Simulering/SimuleringTabell.tsx

```diff
@@ -81,6 +79,16 @@ const LabelMedFarge = styled(Label)`
     color: ${(props: { farge?: string }) => (props.farge ? props.farge : ATextDefault)};
 `;
 
+const StyledSwitch = styled(Switch)`
+    float: right;
```

Comment:
I skissene er switchen venstreplassert, men her har den float right? 

Trenges `TableContainer`? Nå har både den og `StyledTable` propertien `bredde`. Eks så kan man ju (kanskje?) slette bredde fra tablecontainer, eller legge til en wrapper runt switch 
```typescript
const Høyrejustert = styled.div`
    width: ${(props: { bredde: number }) => props.bredde}rem;
    display: flex;
    justify-content: end;
`;
```

Reply:
Anbefaler nog også å bruke flex table i steder for float, for å unngå eventuelle quirks med den 👀

Reply:
Hadde nok oppdatert skissene uten å pushe opp koden 😅 Bruker ikke lenger float, og heller ikke `TableContainer`.